src/electron/get_electron_build_parameters.mjs

```diff
@@ -52,5 +52,5 @@ export function getElectronBuildParameters(parameters) {
     }
   }
 
-  return {platform, buildMode, stagingPercentage, sentryDsn, publish};
+  return {platform, buildMode, publish, sentryDsn, versionName};
```

Comment:
Perhaps rename `publish`? It sounds like a verb or boolean.

Could you clarify what that JSON is in the name? Is that defined somewhere? Perhaps link to the definition in the Electron documentation?